### PR TITLE
Fixing Intermittent Build Errors

### PR DESCRIPTION
**Pull request by HemanthDogiparthi12**

### CODE REVIEW

File: resources/sdk/purecloudjavascript/scripts/test/index.js [hunk 1]
```diff
@@ -102,6 +103,10 @@ describe('JS SDK for Node', function () {
 				done();
 			})
 			.catch((err) => handleError(err, done));
+		}, 6000);
+
+		console.log('after wait');
```
Comment (timsmithgenesys) [inline resources/sdk/purecloudjavascript/scripts/test/index.js:108]: remove debugging trace

**Review by timsmithgenesys — CHANGES_REQUESTED** [review]

File: resources/sdk/purecloudjavascript/scripts/test/index.js [hunk 2]
```diff
@@ -91,7 +91,8 @@ describe('JS SDK for Node', function () {
 	});
 
 	it('should get the user', (done) => {
-		usersApi
+		setTimeout(() => {
```
[the remark this answers is not in the document]
Comment (HemanthDogiparthi12) [reply resources/sdk/purecloudjavascript/scripts/test/index.js:94]: Yes the test waits for the async to complete before marking it as passed. Verified this in local.

The errors we are seeing are more related to the profileSkills not being persisted by the time getUser API call happens.
The settimeout function waits for the configured delay and then executes the getUser call which gives us the necessary buy-in time ( I assume this here) for the profileSkills to be persisted and the asserts to pass.

But as you suggested we can do a retry of the call and see how the builds look. Will update the patch and send it.

[the remark this answers is not in the document]
Comment (carnellj-genesys) [reply resources/sdk/purecloudjavascript/scripts/test/index.js:94]: Good first pass.  I would try to implement the retry (No harm in doing it) and then we can see if gets things to quiet down.

**Review by carnellj-genesys — APPROVED** [review]

File: resources/sdk/purecloudjavascript/scripts/test/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (HemanthDogiparthi12) [reply resources/sdk/purecloudjavascript/scripts/test/index.js:98]: Added retry for Python, Js and .NET SDKs

File: resources/sdk/purecloudpython/scripts/compile.sh [hunk 3]
```diff
@@ -25,5 +25,7 @@ echo "Install six..."
 python3.6 -m pip install --user -U six
 echo "Install watchdog..."
 python3.6 -m pip install --user -U watchdog
+echo "Install watchdog..."
```
Comment (timsmithgenesys) [inline resources/sdk/purecloudpython/scripts/compile.sh:28]: update comment (retry not watchdog)

**Review by timsmithgenesys — APPROVED** [review]

**Pull request merged**